tasks/config/run.js

```diff
@@ -59,6 +74,34 @@ module.exports = function (grunt) {
       ]
     },
 
+    seleniumServer: {
```

Comment:
I'm fine with this, especially in this case, its better to be consistent and understandable than concise.